pallets/author-slot-filter/src/lib.rs

```diff
@@ -259,10 +288,13 @@ pub mod tests {
 	#[test]
 	fn set_eligibility_works() {
 		new_test_ext().execute_with(|| {
-			let percent = Percent::from_percent(34);
+			let value = num::NonZeroU32::new(34);
 
-			assert_ok!(AuthorSlotFilter::set_eligible(Origin::root(), percent));
-			assert_eq!(AuthorSlotFilter::eligible_ratio(), percent)
+			assert_ok!(AuthorSlotFilter::set_eligible(
+				Origin::root(),
+				value.clone()
+			));
+			assert_eq!(AuthorSlotFilter::eligible_count(), value)
 		});
```

Comment:
Can you add one more test which tests the migration? You can take https://github.com/PureStake/moonbeam/blob/c582e2f34a2271a22e60a73dbc6d4ffde90de837/pallets/xcm-transactor/src/tests.rs#L319 as a reference

Reply:
added, please review.